app/statistics/server/lib/statistics.ts

```diff
@@ -449,15 +449,32 @@ export const statistics = {
 			}),
 		);
 
-		statsPms.push(Analytics.resetSeatRequestCount());
+		statsPms.push(
+			new Promise<void>((resolve) => {
+				statistics.totalOTRRooms = Rooms.find({ createdOTR: true }).count();
```

Comment:
the field `createdOTR` is missing an index and this query should be inside the model.. the method can be named liked `Rooms.findByCreatedOTR()`